app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (
+    VaccineError,
+    NotWearingMaskError
+)
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+    masks_to_buy = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError as e:
+            print(e)
+            return "All friends should be vaccinated"
```

Comment:
Consider using a more descriptive message that clearly states the reason for the return, such as 'Not all friends are vaccinated, visit cannot proceed.'